content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -31,9 +41,10 @@ Search for QuestionnaireResponses that meet supplied query parameters:
 
 ### Parameters
 
- Name      | Required? | Type          | Description
------------|-----------|---------------|--------------------------------------------------------
- `_id`     | Yes       | [`token`]     | The logical resource id associated with the resource.
+ Name      | Required?     | Type          | Description
+-----------|---------------|---------------|--------------------------------------------------------------------------------------------
+ `_id`     | Yes           | [`token`]     | The logical resource id associated with the resource.
```

Comment:
This or `patient`

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/65adec4f91ff2c9cdfe7b2a70c0d99e1e5688e42